lighthouse-core/test/lib/asset-saver-test.js

```diff
@@ -89,14 +89,37 @@ describe('asset-saver helper', () => {
       fs.unlinkSync(traceFilename);
     });
 
+    it('prints traces with an event per line', async () => {
+      const trace = {
+        traceEvents: [
+          {args: {}, cat: 'devtools.timeline', pid: 1, ts: 2},
+          {args: {}, cat: 'v8', pid: 1, ts: 3},
+          {args: {IsMainFrame: true}, cat: 'v8', pid: 1, ts: 5},
+          {args: {data: {encodedDataLength: 20, requestId: '1.22'}}, pid: 1, ts: 6},
+        ],
+      };
+      await assetSaver.saveTrace(trace, traceFilename);
+
+      const traceFileContents = fs.readFileSync(traceFilename, 'utf8');
+      expect(traceFileContents).toEqual(
+`{
+"traceEvents": [
+  {"args":{},"cat":"devtools.timeline","pid":1,"ts":2},
+  {"args":{},"cat":"v8","pid":1,"ts":3},
+  {"args":{"IsMainFrame":true},"cat":"v8","pid":1,"ts":5},
+  {"args":{"data":{"encodedDataLength":20,"requestId":"1.22"}},"pid":1,"ts":6}
+]}
+`);
+    });
```

Comment:
we didn't have any tests of the printing style, just that all the correct content was being printed in a JSON.parse()able way.